ansibotmini.py

```diff
@@ -363,6 +363,9 @@
 last_commit: commits(last: 1) {
   nodes {
     commit {
+      statusCheckRollup {
+        state
+      }
```

Comment:
@mkrizek does this return a combined status that GH pre-computes from all the Checks API and Statuses API? I imagine it would have the same weakness. Instead, I recommend trying to specifically look into the Checks API, figuring out the Check Suite for AZP, listing specific Check Run statuses, retrieving the status from the check called `CI` (which is what AZP computes on its side) and relying on that thing.